src/integrations/diesel.rs

```diff
@@ -38,18 +38,23 @@ impl Migration for BarrelMigration {
 
 /// Generate migration files using the barrel schema builder
 pub fn generate_initial(path: &PathBuf) {
+    generate_initial_with_content(path, &"fn up(migr: &mut Migration) {} \n\n".to_string(), &"fn down(migr: &mut Migration) {} \n".to_string())
```

Comment:
Did you let rustfmt run over this before? I'm thinking this could be a lot nicer when it's split across multiple lines :)

Reply:
Thank you, and fixed.